From 68f4b4f5b220f256b17ad2f084d61fe89f3632ae Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Tue, 19 Apr 2016 08:38:38 -0300
Subject: [PATCH] nat: fix build failure

_GNU_SOURCE needs to be defined so that O_CLOEXEC symbols can be found in
system header files.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
Status: submitted upstream

 src/nat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nat.c b/src/nat.c
index 33ae6df..62e21d4 100644
--- a/src/nat.c
+++ b/src/nat.c
@@ -24,6 +24,7 @@
 #include <config.h>
 #endif
 
+#define _GNU_SOURCE
 #include <errno.h>
 #include <sys/types.h>
 #include <sys/stat.h>
-- 
2.7.3

